app/models/url.rb

```diff
@@ -1,9 +1,12 @@
 class Url < ActiveRecord::Base
+  require 'securerandom'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.